src/routes/router.jsx

```diff
@@ -16,15 +16,18 @@ const router = createBrowserRouter([
 
       {
         path: QUESTION_PAGE,
-        element: <QuestionPage />,
+        element: <QuestionPage />, //{"questionCategory/:categoryId/", element: <QuestionPage />},
       },
       {
         path: LOGIN,
         element: <Login />,
       },
+         {
+        path: MENU,
```

Comment:
we do not nmeed a route called Menu. The menu should be on the header only

Reply:
done